proto/ibc/applications/fee/v1/tx.proto

```diff
@@ -15,10 +15,14 @@ service Msg {
   // destination chain must send back relayer's source address (counterparty address) in acknowledgement. This function
   // may be called more than once by a relayer, in which case, latest counterparty address is always used.
   rpc RegisterCounterpartyAddress(MsgRegisterCounterpartyAddress) returns (MsgRegisterCounterpartyAddressResponse);
-  // EscrowPacketFee defines a rpc handler method for MsgEscrowPacketFee
-  // EscrowPacketFee is an open callback that may be called by any module/user that wishes to escrow funds in order to
-  // incentivize the relaying of the given packet.
-  rpc EscrowPacketFee(MsgEscrowPacketFee) returns (MsgEscrowPacketFeeResponse);
+  // PayPacketFee defines a rpc handler method for MsgPayPacketFee
+  // PayPacketFee is an open callback that may be called by any module/user that wishes to escrow funds in order to
+  // incentivize the relaying of the packet at the next sequence
+  rpc PayPacketFee(MsgPayPacketFee) returns (MsgPayPacketFeeResponse);
```

Comment:
I updated the naming here to align with the ticket. From the user's perspective, we are trying to Pay for a packet fee so I think the naming makes sense.

Open to changing this back to `Escrow`